packages/hardhat-plugin-viem/scripts/compile-test-fixture-projects.ts

```diff
@@ -18,10 +18,6 @@ const main = async () => {
       fixtureProjectName
     );
 
-    if (fs.existsSync(path.join(fixtureProjectDir, "./artifacts"))) {
```

Comment:
Was this causing an issue?

The intent was that you only need to do a precompile of the fixture projects to get the viem package build working once - to generate the contract type information. Is there a case I didn't understand?

Reply:
Out of band we discussed and agreed to remove the line.

If you are making changes to the viem tests this can lead to odd results.